README.md

```diff
@@ -133,4 +133,92 @@ TODO
 
 ## Setting up RTC
 
-TODO
+For RPi's not connected to the internet a suitable battery powered hardware
+battery powered hardware clock will be required.
+
+Install clock overhanging the board farthest away from the USB plugs.Follow
+directions on the website exactly.
```

Comment:
I am not sure it is clear what website you are referring to.  It is perfectly fine to add a link to the website here so that people can just go there and get those instructions there.  The risk is of course that the website disappears or goes out of date but we can accept that risk for now.